app/helpers/importHelpers.php

```diff
@@ -1316,15 +1316,15 @@ function caPhpExcelCountNonEmptyRows($ps_xlsx,$ps_sheet=null) {
 	# ---------------------------------------------------------------------
 	/**
 	 * Get content from cell as trimmed string
-	 * @param PHPExcel_Worksheet $po_sheet
+	 * @param \PhpOffice\PhpSpreadsheet\Worksheet\Worksheet $po_sheet
 	 * @param int $pn_row_num row number (zero indexed)
 	 * @param string|int $pm_col either column number (zero indexed) or column letter ('A', 'BC')
-	 * @throws PHPExcel_Exception
+	 * @throws \PhpOffice\PhpSpreadsheet\Exception
 	 * @return string the trimmed cell content
 	 */
 	function caPhpExcelGetCellContentAsString($po_sheet, $pn_row_num, $pm_col) {
 		if(!is_numeric($pm_col)) {
-			$pm_col = PHPExcel_Cell::columnIndexFromString($pm_col)-1;
+			$pm_col = \PhpOffice\PhpSpreadsheet\Cell\Coordinate::columnIndexFromString($pm_col)-1;
```

Comment:
In PHPSpreadsheet coordinates are 1 based, so the code should read

```suggestion
			$pm_col = \PhpOffice\PhpSpreadsheet\Cell\Coordinate::columnIndexFromString($pm_col);
```